frontend/src/main.js

```diff
@@ -12,8 +13,12 @@ Vue.config.devtools = true
 Vue.component('LoadingSpinner', LoadingSpinner)
 Vue.component('AppLogo', AppLogo)
 
-new Vue({
+// read this on a blog == basically if the auth doesn't get ready in time
```

Comment:
Is this comment still relevant?

Reply:
Imma leave it htere for now in case it I need to remind myself how to undo a few other things. I'll remove when I do more work on the auth flow. Turned out that was adding time to app boot. Tradeoff, User will see their logged in status change in first 0.2ms of app.